src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/entity/Offence.kt

```diff
@@ -17,14 +20,16 @@ data class Offence(
   val code: String,
   val description: String? = null,
   val cjsTitle: String? = null,
-  val revisionId: Int? = null,
-  val startDate: LocalDate? = null,
+  val revisionId: Int,
+  val startDate: LocalDate,
   val endDate: LocalDate? = null,
   val category: Int? = null,
   val subCategory: Int? = null,
   val actsAndSections: String? = null,
   val parentOffenceId: Long? = null,
-  val changedDate: LocalDateTime? = null,
+  @Enumerated(EnumType.STRING)
+  val sdrsCache: SdrsCache,
+  val changedDate: LocalDateTime,
```

Comment:
I made several fields mandatory based on the data we get from SDRS.. previously we were tentative about this because we weren't sure about the quality of data from SDRS